### PR TITLE
Incident custom fields

### PR DESCRIPTION
**Pull request by glicht**
## Status
Ready

## Related Issues
fixes: #50

## Description
* Add support for CommonFields in incident model
* add verify.sh to build which checks that code isn't generated by code-gen.sh
* stop testing on python 3.6 and test on 3.8

## Must have
- [ ] Unit Test or Example Code
- [ ] Changelog entry

### CODE REVIEW

File: demisto_client/demisto_api/api/default_api.py [hunk 1]
```diff
@@ -3796,25 +3797,22 @@ def get_widget_with_http_info(self, id, **kwargs):  # noqa: E501
             _request_timeout=params.get('_request_timeout'),
             collection_formats=collection_formats)
 
-    def import_classifier(self, file, **kwargs):  # noqa: E501
+    def import_classifier(self, file, classifier_id, **kwargs):  # noqa: E501
```
Comment (glicht) [inline demisto_client/demisto_api/api/default_api.py:3800]: @gal-berger please review. Not clear why this change wasn't included in your PR.

Comment (gal-berger) [reply demisto_client/demisto_api/api/default_api.py:3800]: I think it should be like it is on master because this is the most updated version. I think that someone probably forgot to add it to gen-code.
it is my bad for not added it before into my PR, I thought I've fixed all the diff that was between master and running gen-code that wasn't related to my code.
so we need to add this lines to gen-code to fix it:
`sed -i '' -e 's/def import_classifier(self, file, classifier_id, \*\*kwargs):  # noqa: E501/def import_classifier(self, file, \*\*kwargs):  # noqa: E501/' demisto_client/demisto_api/api/default_api.py
`
`sed -i '' -e 's/>>> thread = api.import_classifier(file, classifier_id, async_req=True)/>>> thread = api.import_classifier(file, async_req=True)/' demisto_client/demisto_api/api/default_api.py`

Comment (glicht) [reply demisto_client/demisto_api/api/default_api.py:3800]: @ShahafBenYakir why do we have this change?

**Review by gal-berger — APPROVED** [review]
finish adding import_classifier changes and update docs as well.
